Makefile

```diff
@@ -31,3 +31,12 @@ clean:
 # install required dependencies
 install-deps:
 	$(MAKE) -C $(PROTODIR) install-deps
+
+# install and run golangci-lint. sudo required
+lint:
+	@echo "install golangci-lint"
+	@curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.31.0
+	@golangci-lint --version
+	@echo "lint degitx"
+	@golangci-lint run
+	@echo "degitx linted"
```

Comment:
@stepanvalyavskiy you can just call `golangci-lint run`: `@` symbol is hiding command statement, it's useful for `@echo` when you don't need to show echo command itself. But `golangci-lint run` itself is verbose enough and can be printed to output as is. So all these `@echo` commands will not be needed when printing lint to output